### PR TITLE
Correct APT::0x00550040 and APT::0x00560000 function

### PR DESCRIPTION
**Pull request by JamePeng**
Some guy corrected the old and confusing description, so I need to fix my old wrong code :smiley: 

https://www.3dbrew.org/wiki/APT:GetScreencapPostPermission
https://www.3dbrew.org/wiki/APT:SetScreencapPostPermission

### CODE REVIEW

File: src/core/hle/service/apt/apt.cpp [hunk 1]
```diff
@@ -35,7 +35,9 @@ static Kernel::SharedPtr<Kernel::Event> parameter_event; ///< APT parameter even
 static u32 cpu_percent; ///< CPU time available to the running application
 
 // APT::CheckNew3DSApp will check this unknown_ns_state_field to determine processing mode
-static u8 unknown_ns_state_field;
+static u8 unknown_ns_state_field = 0;
```
Comment (JayFoxRox) [inline src/core/hle/service/apt/apt.cpp]: This is already initialized in `Init`?

I'm guessing this should be set to zero on every reboot of the virtual 3DS?
If so the initialization here is probably useless and should be done elsewhere.

File: src/core/hle/service/apt/apt.h [hunk 2]
```diff
@@ -94,6 +94,13 @@ enum class StartupArgumentType : u32 {
     OtherMedia = 2,
 };
 
+enum class ScreencapPostPermission : u32 {
+    CleanThePermission                 = 0,
```
Comment (Subv) [inline src/core/hle/service/apt/apt.h]: What does CleanThePermission mean?

Comment (JamePeng) [inline src/core/hle/service/apt/apt.h]: qq 20160715004024

The SetScreencapPostPermission first call will set to zero

Comment (Subv) [inline src/core/hle/service/apt/apt.h]: Yeah but what does it mean? And how is it different than screen_capture_post_permission=1?
That needs to be either documented in the code or left as a todo.

Comment (JamePeng) [inline src/core/hle/service/apt/apt.h]: Add a todo will be better to notice this enum value, but we don't know what is the inital value it is, and the zero will be more safer than [screen_capture_post_permission=1].

**Comment by bunnei** [comment]
LGTM

**Pull request merged**